### DISCUSSION TITLE
Example visualizations

### DISCUSSION THREAD

**Discussion by romain-fontugne** [InternetHealthReport/gsoc#15] [Ideas]
These are inspiring examples for this project:
- https://explorer.ooni.org
- https://ioda.inetintel.cc.gatech.edu/dashboard
- https://lookerstudio.google.com/s/heX1h7lGKag

**Comment by moawnallah** [comment]
These are really good examples, Thanks for sharing!

**Comment by romain-fontugne** [comment]
oh and I forgot Cloudflare's radar: https://radar.cloudflare.com/as2497
